rasa/shared/core/flows/flow.py

```diff
@@ -257,13 +254,20 @@ def step_for_id(self, step_id: Optional[Text]) -> Optional[FlowStep]:
         if not step_id:
             return None
 
+        if step_id == START_STEP:
+            first_step_in_flow = self.first_step_in_flow()
+            return StartFlowStep(first_step_in_flow.id if first_step_in_flow else None)
+
+        if step_id == END_STEP:
+            return EndFlowStep()
+
         for step in self.steps:
             if step.id == step_id:
                 return step
 
         return None
 
-    def start_step(self) -> Optional[FlowStep]:
+    def first_step_in_flow(self) -> Optional[FlowStep]:
```

Comment:
Are there any kinds of first steps we expect for flows? Just a nitpick, maybe `first_step` is sufficient.